home/filter.go

```diff
@@ -478,6 +478,8 @@ func (f *Filtering) parseFilterContents(file io.Reader) (int, uint32, string) {
 				name = m[0][1]
 				seenTitle = true
 			}
+		} else if line[0] == '#' {
+			continue			
```

Comment:
I think the Travis build is failing because you've left the trailing white space after `continue`.  Can you remove it?

Reply:
@szolin it looks like that did it, thank you.